docs/features/change_granule_collection.md

```diff
@@ -0,0 +1,91 @@
+
+### concurrency
+
+This configuration defines parallelization to use. Defaults to 100.
+
+The specific subroutines that run at this concurrency are:
+
+- loading granule records from the cumulus Api
+- copying s3 files to new location (if necessary)
+- deleting old s3 files (if necessary)
+- updating granule records in cumulus datastores (es/postgres)
+- updating records in cmr
+
+### dbMaxPool
+
+This configuration specifies how many database connections to allow the process to utilize as part of it's connection pool. This value will constrain database connections, but too low a value can cause performance issues or database write failures (Knex timeout errors) if the connection pool is not high enough to support the set concurrency.   Defaults 100, value should target at minimum the value set for `concurrency`.
+
+### maxRequestGranules
+
+This configuration limits the size of requests sent to api endpoints during the runtime of this workflow. Defaults to 1000, minimize overhead from api calls by setting higher, but must be small enough that an api call (primarily the stringified set of granules) does not overrun the limit of 6 Mb that can be passed, and also wont time out the api lambda in attempting to run. how this should be set depends heavily on how many files there are per granule, and what the privateApiLambda timeout is set to
+
+the specific subroutines that run this maxRequestGranules are
+
+- `bulkPatch` granule api endpoint
+- `bulkPatchGranuleCollection` granule api endpoint
+
+### invalidGranuleBehavior
+
+This configuration specifies how to handle granule records that are un-processable due to containing file records missing either bucket or key. Defaults to `error`, can be either `error` or `skip`. It is of course hoped that these don't exist, but if they do show up it is, by default, expected that this should be taken as a needed fix. if however you wish to process what can be processed and come back to the rest later, they can be skipped
+
+### cmrGranuleUrlType
+
+This specifies what type of urls to fill out in the cmr metadata as it is updated to the new collection's pathing pattern. Defaults to `both`, can be `both`, `s3`, or `http`.
+
+### s3MultipartChunkSizeMb
+
+This allows you to set the size of chunks that files should be broken up into when loading to s3. This should, in most cases, be left unset, to use the value in your environment based upon the same global configuration that sets this same value elsewhere (i.e. move-granules task) in accordance with your stack's configuration
+
+## Implementation
+
+The intended use of this is to call this api on a rhythm untill it is done. this could be done with a cron job, or another script, but the same api call can be made again and again and by the way it is structured it will get new granules, or reprocess old granules that failed to process the first time.
```

Comment:
`untill` has one too many `l`s
also, sentences after periods (and on new lines) should be capitalized.